Sources/JXKit/ScriptManager.swift

```diff
@@ -1,4 +1,4 @@
-import struct Foundation.URL
+import Foundation
```

Comment:
Lots of internal updates here that don't really matter externally, except to simplify our other code a little.